src/databricks/labs/ucx/cli.py

```diff
@@ -141,7 +141,8 @@ def revert_migrated_tables(schema: str, table: str, *, delete_managed: bool = Fa
     warehouse_id = installation.config.warehouse_id
     sql_backend = StatementExecutionBackend(ws, warehouse_id)
     table_crawler = TablesCrawler(sql_backend, installation.config.inventory_database)
-    tm = TablesMigrate(table_crawler, ws, sql_backend)
+    tmp = TableMapping(ws)
```

Comment:
use longer names :) tm, tmp, tmpl, tplm... gets harder to read once you start using them inconsistently

Reply:
OK